flag/flag.go

```diff
@@ -3,11 +3,13 @@ package flag
 import (
 	"github.com/giantswarm/microkit/flag"
 
+	"github.com/giantswarm/cluster-operator/flag/guest"
```

Comment:
Whole `flag/guest` structure is copied from another design PR as-is. Most of it is not used right now, but during development of this it felt that it's easier to work with `cluster-operator` code if the whole structure is present and when `AWSClusterConfig` and `KVMClusterConfig` are implemented, then remove unused parts, if they still exist.